.gitignore

```diff
@@ -19,6 +19,7 @@ build/*
 SampleApps/AEPIdentityEdgeSampleApp/build
 ci/*
 out/*
+TestProject/*
```

Comment:
This does not seem to be a known path. Is this something you have locally and can be removed?

Reply:
This is created while we run make `test-podspec`. So I wanted to avoid any confusion of it being added if we test those scripts and commit changes in future.